src/styles/__stars.scss

```diff
@@ -0,0 +1,21 @@
+.card__stars {
+  display: flex;
+
+  &-item {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+
+  }
+
+  &--1 .card__stars-item:nth-child(-n+1),
+  &--2 .card__stars-item:nth-child(-n+2),
+  &--3 .card__stars-item:nth-child(-n+3),
+  &--4 .card__stars-item:nth-child(-n+4),
+  &--5 .card__stars-item:nth-child(-n+5) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
please use loops for things like that: checklist [point ](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)12